raiden/messages/metadata.py

```diff
@@ -1,52 +1,39 @@
 from copy import deepcopy
 from dataclasses import dataclass
+from typing import Any
 
 import canonicaljson
-from eth_utils import keccak
+from eth_utils import keccak, to_checksum_address
+from marshmallow import EXCLUDE, post_dump, post_load
 
 from raiden.messages.abstract import cached_property
-from raiden.utils.formatting import to_checksum_address
-from raiden.utils.typing import MYPY_ANNOTATION, Address, AddressMetadata, Dict, List, Optional
+from raiden.storage.serialization import serializer
+from raiden.utils.typing import Address, AddressMetadata, Dict, List, Optional
 from raiden.utils.validation import MetadataValidation
 
 
 @dataclass
 class RouteMetadata(MetadataValidation):
     route: List[Address]
-    address_metadata: Optional[Dict[Address, AddressMetadata]]
+    address_metadata: Optional[Dict[Address, AddressMetadata]] = None
 
-    def __init__(
+    class Meta:
```

Comment:
Can you add a docstring to state what the internal Meta class is used for?

Reply:
I now referenced the class where the custom option is defined and referenced links from their docs where to find more about this rather special mechanism in marshmallow schemas.